week04/ecommerce/Product_Detail.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Detail</title>
+    <link rel="stylesheet" media="screen and (max-width: 800px)" href="smallstyle.css">
+    <link rel="stylesheet" media="screen and (min-width: 800px)" href="fullstyle.css">
+</head>
+<header>
+    <!-- Navbar -->
+    <nav class="navbar">
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="products.html">Products</a></li>
+            <li><a href="contact.html">Contact</a></li>
+        </ul>
+    </nav>
+</header>
+
+<body>
+
+    <h1>Product Details</h1>
+
+    <div> <img src="images/golf_set.jpeg" alt="Golf Set"></div>
```

Comment:
I'm assuming you wrapped the image in a div to get the image to drop to a new line? In this case i would instead adjust your "display" attribute on your image.